src/__Private/LintRunCLIEventHandler.hack

```diff
@@ -70,13 +70,13 @@ final class LintRunCLIEventHandler implements LintRunEventHandler {
           $error->getDescription(),
           $colors ? "\e[0m" : '',
           $colors ? "\e[90m" : '',
-          \get_class($error->getLinter()),
+          \get_class($error->getLintRule()),
```

Comment:
- For hh_client --lint, the class name of the lint rule is not useful; I think you want $error->getLintRule()->getName() here?
- given this use case, I think we should split up 'human-readable lint rule name' from 'code'; for example, while  I think we want `HHAST_IGNORE_ERROR[1234]`, I don't think we want the CLI to say `Linter: 1234` - maybe `Linter: Hack Lint 1234`.